src/main/java/io/curity/identityserver/plugin/events/listeners/config/AWSEventListenerConfiguration.java

```diff
@@ -71,4 +71,24 @@
     }
 
     ExceptionFactory getExceptionFactory();
+
+    @Description("Configure the hashing algorithm that will be used to hash the signature of the split token.")
```

Comment:
shouldn't we set a default using `se.curity.identityserver.sdk.config.annotation.DefaultEnum`?